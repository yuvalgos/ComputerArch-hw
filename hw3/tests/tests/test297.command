./dflow_calc tests/test297.dat tests/test297.in p33 p120 d104 d97 d111 d165 p64 p119 d41 d147 p31 p127 p123 d148 d182 d120 d65 d61 p45 d103 d56 p157 p0 d78 p7 d0 p20 d84 p95 p178 d138 d110 p41 d183 d27 d178 d121 d149 d109 p60 d153 d167 d124 p36 p54 d4 d5 d166 d174 p82 d76 p11 p149 p65 p149 p37 d193 p114 d78 p125 p66 p93 d60 p104 d123 p45 d85 p60 d119 p24 p2 d120 p113 p169 d165 p67 p120 d54 d171 d138 d107 p140 d180 d16 p163 d147 p132 d42 p72 p136 d128 p35 d3 p106 d111 d181 p69 p159 p53 p128 p154 d131 d51 d38 p23 p159 d81 d146 p6 p18 d117 p4 d131 p71 d18 p66 p4 d143 d90 d84 d122 d83 p173 p188 d21 p57 p6 p128 p114 d187 p65 p43 p187 p5 p64 d64 d69 d48 p138 p2 p134 p178