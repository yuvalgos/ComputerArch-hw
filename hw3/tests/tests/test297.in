3 27 2 24
5 14 25 19
4 8 0 3
8 2 21 19
0 6 1 10
7 26 3 31
0 3 27 16
2 28 18 23
6 14 16 6
2 16 26 22
2 30 27 8
4 9 7 7
8 12 28 10
3 10 31 7
8 0 15 3
5 19 21 20
4 2 14 16
4 17 5 26
4 1 12 29
0 14 12 19
8 2 29 0
4 22 14 12
0 17 26 3
5 4 31 24
8 10 27 15
2 7 30 14
4 5 11 20
2 13 26 31
6 9 20 6
0 7 6 0
4 26 19 14
6 10 11 6
3 9 2 21
7 19 24 25
6 3 30 9
6 29 5 30
6 25 21 0
2 18 1 11
0 12 28 0
8 22 26 14
4 19 14 19
2 9 26 18
2 3 11 20
7 22 7 24
1 15 15 24
4 21 27 17
3 4 14 18
5 16 18 13
5 29 19 31
8 21 20 23
8 8 1 9
1 21 24 31
7 23 2 12
1 28 0 14
4 2 9 10
3 16 26 16
5 25 12 4
0 19 1 31
6 0 15 13
0 14 31 25
3 11 5 17
8 2 17 9
4 27 10 23
6 12 12 16
2 15 6 26
6 11 17 21
4 25 12 25
3 24 27 2
6 16 24 17
6 17 2 25
8 5 20 14
2 27 14 31
8 13 16 3
3 0 5 18
7 2 1 8
4 21 29 2
4 15 13 7
2 30 25 3
3 18 11 18
3 3 4 6
8 12 3 2
5 13 17 17
1 11 8 23
4 28 8 24
4 21 23 30
6 11 18 12
2 23 28 22
2 31 13 18
8 5 0 1
3 11 27 24
7 29 1 17
4 29 6 18
7 1 13 6
2 24 18 18
2 8 15 22
3 15 25 24
0 28 6 30
5 28 6 20
6 24 12 22
0 8 2 19
4 18 3 16
5 26 24 0
6 7 5 19
2 8 23 11
0 20 27 21
7 12 12 0
1 11 6 24
7 6 20 30
5 23 29 22
3 3 10 27
3 3 6 2
5 18 9 17
7 29 28 13
2 6 10 14
2 1 23 22
0 4 14 5
1 25 8 17
5 14 9 19
4 29 19 2
2 3 13 30
1 20 22 26
6 11 3 8
8 24 6 23
3 14 14 24
3 9 3 19
3 1 30 11
0 10 8 28
2 2 29 21
5 18 27 0
6 10 31 0
4 7 7 21
2 19 9 24
3 30 11 26
0 19 28 29
4 12 24 6
1 14 11 12
3 1 22 23
7 25 13 16
3 30 0 11
8 29 13 28
3 2 4 25
8 25 13 28
7 26 25 25
1 5 1 7
3 2 19 7
2 8 6 11
7 1 22 26
8 4 1 9
4 15 9 21
0 6 0 17
3 19 0 25
7 23 10 20
7 0 18 9
6 21 2 4
4 0 0 14
2 3 6 25
8 11 26 29
0 19 10 21
2 6 21 27
4 28 6 6
6 6 18 9
1 28 21 22
5 9 9 8
6 13 13 21
5 3 18 1
5 1 26 12
5 7 11 31
1 21 31 8
4 7 7 24
1 24 1 22
3 19 18 21
5 28 20 30
2 17 26 22
4 26 4 14
1 7 16 23
0 12 29 3
2 1 9 25
8 22 18 5
7 9 25 26
1 6 10 4
7 29 16 20
4 15 9 25
7 7 25 16
3 14 14 12
4 9 14 12
2 10 18 16
5 24 23 27
4 17 2 11
4 22 0 19
5 25 26 7
3 13 7 4
4 11 4 3
6 8 18 27
7 3 9 17
0 10 3 11
0 3 20 18
5 4 12 19
7 16 12 12
2 6 5 20
5 25 16 25
