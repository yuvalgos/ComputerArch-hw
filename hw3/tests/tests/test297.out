Reading the opcodes latency info from tests/test297.dat ... Got latency for 9 opcodes
Reading the program file tests/test297.in ... Found 200 instructions
getProgDepth()==247
getDepDepth(33)==0
getDepDepth(120)==144
getInstDeps(104)=={71,84}
getInstDeps(97)=={4,-1}
getInstDeps(111)=={41,69}
getInstDeps(165)=={142,134}
getDepDepth(64)==41
getDepDepth(119)==122
getInstDeps(41)=={30,37}
getInstDeps(147)=={146,124}
getDepDepth(31)==2
getDepDepth(127)==108
getDepDepth(123)==98
getInstDeps(148)=={124,84}
getInstDeps(182)=={141,68}
getInstDeps(120)=={43,101}
getInstDeps(65)=={22,51}
getInstDeps(61)=={22,41}
getDepDepth(45)==21
getInstDeps(103)=={86,89}
getInstDeps(56)=={38,46}
getDepDepth(157)==223
getDepDepth(0)==0
getInstDeps(78)=={65,37}
getDepDepth(7)==0
getInstDeps(0)=={-1,-1}
getDepDepth(20)==32
getInstDeps(84)=={52,77}
getDepDepth(95)==92
getDepDepth(178)==217
getInstDeps(138)=={73,121}
getInstDeps(110)=={107,74}
getDepDepth(41)==44
getInstDeps(183)=={135,175}
getInstDeps(27)=={5,-1}
getInstDeps(178)=={141,173}
getInstDeps(121)=={119,103}
getInstDeps(149)=={73,69}
getInstDeps(109)=={31,71}
getDepDepth(60)==21
getInstDeps(153)=={144,147}
getInstDeps(167)=={87,145}
getInstDeps(124)=={119,57}
getDepDepth(36)==32
getDepDepth(54)==49
getInstDeps(4)=={-1,-1}
getInstDeps(5)=={-1,-1}
getInstDeps(166)=={156,87}
getInstDeps(174)=={68,151}
getDepDepth(82)==74
getInstDeps(76)=={72,29}
getDepDepth(11)==0
getDepDepth(149)==109
getDepDepth(65)==21
getDepDepth(149)==109
getDepDepth(37)==39
getInstDeps(193)=={184,187}
getDepDepth(114)==75
getInstDeps(78)=={65,37}
getDepDepth(125)==144
getDepDepth(66)==81
getDepDepth(93)==51
getInstDeps(60)=={26,22}
getDepDepth(104)==108
getInstDeps(123)=={117,122}
getDepDepth(45)==21
getInstDeps(85)=={78,80}
getDepDepth(60)==21
getInstDeps(119)=={81,77}
getDepDepth(24)==7
getDepDepth(2)==0
getInstDeps(120)=={43,101}
getDepDepth(113)==66
getDepDepth(169)==177
getInstDeps(165)=={142,134}
getDepDepth(67)==85
getDepDepth(120)==144
getInstDeps(54)=={41,31}
getInstDeps(171)=={120,138}
getInstDeps(138)=={73,121}
getInstDeps(107)=={104,77}
getDepDepth(140)==141
getInstDeps(180)=={68,120}
getInstDeps(16)=={8,9}
getDepDepth(163)==223
getInstDeps(147)=={146,124}
getDepDepth(132)==144
getInstDeps(42)=={-1,-1}
getDepDepth(72)==109
getDepDepth(136)==75
getInstDeps(128)=={71,73}
getDepDepth(35)==16
getInstDeps(3)=={-1,-1}
getDepDepth(106)==131
getInstDeps(111)=={41,69}
getInstDeps(181)=={178,141}
getDepDepth(69)==92
getDepDepth(159)==228
getDepDepth(53)==32
getDepDepth(128)==96
getDepDepth(154)==153
getInstDeps(131)=={124,122}
getInstDeps(51)=={-1,-1}
getInstDeps(38)=={7,14}
getDepDepth(23)==0
getDepDepth(159)==228
getInstDeps(81)=={69,69}
getInstDeps(146)=={43,142}
getDepDepth(6)==7
getDepDepth(18)==28
getInstDeps(117)=={41,57}
getDepDepth(4)==0
getInstDeps(131)=={124,122}
getDepDepth(71)==66
getInstDeps(18)=={12,-1}
getDepDepth(66)==81
getDepDepth(4)==0
getInstDeps(143)=={136,130}
getInstDeps(90)=={18,69}
getInstDeps(84)=={52,77}
getInstDeps(122)=={113,108}
getInstDeps(83)=={50,67}
getDepDepth(173)==206
getDepDepth(188)==164
getInstDeps(21)=={19,12}
getDepDepth(57)==39
getDepDepth(6)==7
getDepDepth(128)==96
getDepDepth(114)==75
getInstDeps(187)=={144,156}
getDepDepth(65)==21
getDepDepth(43)==34
getDepDepth(187)==187
getDepDepth(5)==0
getDepDepth(64)==41
getInstDeps(64)=={4,30}
getInstDeps(69)=={61,66}
getInstDeps(48)=={40,-1}
getDepDepth(138)==144
getDepDepth(2)==0
getDepDepth(134)==98
getDepDepth(178)==217
